rio_tiler/utils.py

```diff
@@ -384,6 +384,13 @@ def get_vrt_transform(
 
     w, s, e, n = bounds
 
+    if height and width:
+        out_transform = from_bounds(w, s, e, n, width, height)
+        # when not overzooming we don't want to set width/height output
+        if (out_transform.a > dst_transform.a) and (out_transform.e < dst_transform.e):
```

Comment:
This seems to assume that `e` is negative - may need to `abs` the pixel sizes for a safe comparison?